packages/seacas/libraries/ioss/src/CMakeLists.txt

```diff
@@ -74,6 +74,15 @@ IF (MINGW)
   target_link_libraries(Ioss psapi)
 ENDIF ()
 
+TARGET_INCLUDE_DIRECTORIES(
```

Comment:
Is this needed?

Reply:
See https://github.com/gsjaardema/seacas/pull/214#issuecomment-776156905